pkg/pillar/cmd/baseosmgr/baseosmgr.go

```diff
@@ -434,11 +437,10 @@ func handleGlobalConfigModify(ctxArg interface{}, key string,
 		log.Infof("handleGlobalConfigModify: ignoring %s\n", key)
 		return
 	}
-	var gcp *types.GlobalConfig
-	debug, gcp = agentlog.HandleGlobalConfig(ctx.subGlobalConfig, agentName,
-		debugOverride)
+	var gcp *types.ConfigItemValueMap
```

Comment:
Why do we need this?

Reply:
@kalyan-nidumolu Can't do gcp, debug := agentlog... below since it would declare both gcp and debug variables, and we need to use the global deug variable.